packages/special-pages/tests/page-objects/specialerror.js

```diff
@@ -88,8 +90,43 @@ export class SSLErrorPage {
     async visitsSite () {
         const { page } = this
         await page.pause()
-        await page.getByRole('button', { name: 'Advanced...' }).click()
+        this.showsAdvancedInfo()
         await page.getByRole('button', { name: 'Accept Risk and Visit Site' }).click()
         await this.mocks.waitForCallCount({ method: 'visitSite', count: 1 })
     }
+
+    /**
+     * Clicks on help links in Phishing page
+     */
+    async opensPhishingHelpPage () {
+        await this.opensNewPage('Learn more', 'https://duckduckgo.com/duckduckgo-help-pages/')
+        await this.showsAdvancedInfo()
+        await this.opensNewPage('Phishing and Malware Protection help page', 'https://duckduckgo.com/duckduckgo-help-pages/')
+    }
```

Comment:
very nice test 💪

Since these are individual steps taken by the user though, I think you can remove this `opensPhishingHelpPage` method and just put these 3 lines into the spec file directly.